__exekutir__/inventory/host_vars/kommandir.yml

```diff
@@ -87,4 +87,6 @@ git_cache_args:
     - repo: "https://github.com/python-bugzilla/python-bugzilla.git"
       recursive: False
       cachepath: "python-bugzilla"
+      # Do not include a 'depth' argument, git needs the full history to be 
+      # able to locate specific commits/branches/tags referenced by 'version'
```

Comment:
@afomm if the `depth` option were actually documented anywhere, then yes, it would make sense to extend that documentation with this caveat. But (1) it's not documented; and (2) even if someone _does_ include a depth argument, the new code will gracefully ignore it, so there isn't really a need for the warning. I think removing it is the best option.